src/components/MultiSelectGroup/MultiSelectGroup.tsx

```diff
@@ -18,93 +22,77 @@ export interface MultiSelectGroupProps {
+    const {
+      className = "",
+      id,
+      labelText,
+      layout = "row",
+      multiSelectWidth = "full",
+      showLabel = true,
+      renderMultiSelect,
+      ...rest
+    } = props;
+    const { isLargerThanMobile } = useNYPLBreakpoints();
+    const finalLayout = isLargerThanMobile ? layout : "column";
+    const finalWidth = isLargerThanMobile ? multiSelectWidth : "full";
+    const isBlockElement = finalLayout === "column";
 
-      const { isLargerThanMobile } = useNYPLBreakpoints();
-      const finalLayout = isLargerThanMobile ? layout : "column";
-      const finallWidth = isLargerThanMobile ? multiSelectWidth : "full";
-      const isBlockElement = layout === "column" ? true : false;
-      const styles = useMultiStyleConfig("MultiSelectGroup", {
-        width: finallWidth,
-      });
-
-      // Go through the MultiSelect children and update props as needed.
-      React.Children.map(
-        children as JSX.Element,
-        (child: React.ReactElement) => {
-          if (React.isValidElement(child)) {
-            // @TODO: DXP needs to pass custom MultiSelects that wrap DS Mutliselects - type check deos not allow DXP to use MultiSelectGroup
-            // if (child.type !== MultiSelect) {
-            //   console.warn(
-            //     "NYPL Reservoir MultiSelectGroup: Only MultiSelect components can be children of MultiSelectGroup."
-            //   );
-            //   return;
-            // }
-            if (child.type === MultiSelect) {
-              const props = { isBlockElement, width: multiSelectWidth };
-              newChildren.push(React.cloneElement(child, props));
-            } else {
-              newChildren.push(React.cloneElement(child));
-            }
-          }
-        }
-      );
-
-      return (
-        <Fieldset
-          id={`${id}-multiselect-group`}
-          legendText={labelText}
-          isLegendHidden={!showLabel}
-          __css={styles}
-          {...rest}
-        >
-          <Stack
-            className={className}
-            columnGap="xs"
-            data-testid="multi-select-group"
-            direction={finalLayout}
-            id={id}
-            ref={ref}
-            rowGap={finalLayout === "row" ? "xs" : "0"}
-            spacing="xs"
-            wrap={layout === "row" ? "wrap" : null}
-            sx={{
-              "> div": {
-                _notFirst: {
-                  mx: "0",
-                },
+    return (
+      <Fieldset
+        id={`multiselect-group-${id}`}
+        legendText={labelText}
+        isLegendHidden={!showLabel}
+        {...rest}
+      >
+        <Stack
+          className={className}
+          columnGap="xs"
+          data-testid={`multiselect-group-${id}`}
+          direction={finalLayout}
+          id={id}
+          ref={ref}
+          rowGap={finalLayout === "row" ? "xs" : "0"}
+          spacing="xs"
+          wrap={layout === "row" ? "wrap" : null}
```

Comment:
This is what is causing the MultiSelect components to go full width for the row layout. Update the condition to set the value as "wrap" when layout === "row" and multiSelectWidth === "fitContent".